### PR TITLE
[ISSUE #2603]  ContainsPredicateJudge not work for multiple config

### PR DESCRIPTION
**Pull request by MiniSC**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/condition/data/URIParameterData.java [hunk 1]
```diff
@@ -17,17 +17,31 @@
...
-    
+
     @Override
     public String builder(final String paramName, final ServerWebExchange exchange) {
         return exchange.getRequest().getURI().getPath();
     }
+
+    @Override
+    public Boolean containsJudge(final ConditionData conditionData, final String realData) {
+        String regex = ",";
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/condition/data/URIParameterData.java:42]: URI have contains？ why not used match？

Comment (MiniSC) [reply shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/condition/data/URIParameterData.java:42]: Mainly to fix the IP contains problem, this upgrading for uri contains just fit the old test cast in PredicateJudgeFactoryTest and let uri can be contains, but i think this separator [,] will not be used

**Comment by MiniSC** [comment]
I remove my upgrading for uri , restore the old implementation for pass the PredicateJudgeFactoryTest

**Comment by yu199195** [comment]
@MiniSC  hi， uri used match method, whether it overrides the contains？

**Comment by MiniSC** [comment]
> @MiniSC hi， uri used match method, whether it overrides the contains？

because old contains let all param type use realData to contains condition's Data ,  old PredicateJudgeFactoryTest want let condition's uri like /http/** can be matched with /http/**/xxx in realData, but my upgrading will use condition's Data to contains realData, can not pass the old Test,  and i think in the problem of uri contains ,you are right , so i restore the old implements with uri, and it's reason of ovverride the contains

**Comment by yu199195** [comment]
@MiniSC  hi, i mean uri used match, Is it no longer necessary to use the contains
>  because old contains method impelement let all param type use realData to contains condition's Data , old PredicateJudgeFactoryTest want let condition's uri like /http/** can be matched with /http/**/xxx in realData, but my upgrading will use condition's Data to contains realData, can not pass the old Test, and i think in the problem of uri contains ,you are right ,we will not use contains in uri , so i restore the old implements with uri, and it's reason of ovverride the contai

**Comment by MiniSC** [comment]
> @MiniSC hi, i mean uri used match, Is it no longer necessary to use the contains
> 
> > because old contains method impelement let all param type use realData to contains condition's Data , old PredicateJudgeFactoryTest want let condition's uri like /http/** can be matched with /http/**/xxx in realData, but my upgrading will use condition's Data to contains realData, can not pass the old Test, and i think in the problem of uri contains ,you are right ,we will not use contains in uri , so i restore the old implements with uri, and it's reason of ovverride the contai

so you means i should remove the override and the test case?

File: shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/condition/data/ParameterData.java [hunk 2]
```diff
@@ -25,15 +26,28 @@
...
+
+    /**
+     * Judge type for contains,
+     * different Param Type have own comparison method.
+     *
+     * @param conditionData the rule conditionData
+     * @param realData      the request's realData
+     * @return {@link Boolean}
+     */
+    default Boolean containsJudge(final ConditionData conditionData, final String realData) {
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/condition/data/ParameterData.java:50]: remove this

**Review by yu199195 — APPROVED** [review]

**Comment by yu199195** [comment]
@MiniSC  you can add my wechat xixy199195， i will join you to contributor group

**Pull request merged**